lib/elixir/test/elixir/inspect_test.exs

```diff
@@ -154,6 +157,7 @@ defmodule Inspect.NumberTest do
 
   test "float" do
     assert inspect(1.0) == "1.0"
+    assert inspect(1234.001) == "1_234.001"
```

Comment:
Are we adding separators to the fractional part of the float? If not, can we add a test for it?